docs/use-cases/sensor-data-query.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Query sensor data with third-party tools"
+linkTitle: "Query sensor data"
+weight: 30
+type: "docs"
+images: ["/services/icons/data-query.svg"]
+description: "Query sensor data that you have synced to the Viam app using the data management service with SQL or MQL."
+modulescript: true
+aliases:
+  - /manage/data/query/
+  - /data/query/
+# SME: Devin Hilly
+---
+
+You can use the data management service to capture sensor or time-series data from any machine and sync that data to the cloud.
+Then, you can query it using {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} to obtain actionable insights or connect it to third-party visualization tools.
+
+For example, you can configure data capture for several sensors on one machine, or for serveral sensors across multiple machines, to report the ambient operating temperature.
+You can then run queries against that data to search for outliers or edge cases, to analyze how the ambient temperature affects your machines' operation.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Gathering data on any machine and syncing it to the cloud](#gather-and-sync-data).
+1. [Configuring data query](#configure-data-query).
+1. [Querying data from third-party tools](#query-data-using-third-party-tools).
+
+{{< /alert >}}
+
+## Prerequisites
```

Comment:
Why not make the capture data use case a prereq for all three of these?

Reply:
I thought it would be nice to just include it as an include. But yeah can do